spec/team_statistics_spec.rb

```diff
@@ -0,0 +1,15 @@
+require './spec/spec_helper'
+
+RSpec.describe TeamStatistics do 
+  before(:each) do 
+    game_path = './data/games_dummy.csv'
+    team_path = './data/teams_dummy.csv'
+    game_teams_path = './data/game_teams_dummy.csv'
+
+    locations = {
+      games: game_path,
+      teams: team_path,
+      game_teams: game_teams_path
+    }
+    @stat_tracker = StatTracker.from_csv(locations)
+    @team_statistics = TeamStatistics.new(@stat_tracker.teams, @stat_tracker)
```

Comment:
Do we only need more than teams for these stats? looks good otherwise.

Reply:
Good call.  It references seasonal data so I added games as well so we can parse season data.